ca/certificate-authority.go

```diff
@@ -47,6 +49,56 @@ var badSignatureAlgorithms = map[x509.SignatureAlgorithm]bool{
 	x509.ECDSAWithSHA1:             true,
 }
 
+// PKCS#9 attribute for requesting certificate extensions
+var oidExtensionRequest = asn1.ObjectIdentifier{1, 2, 840, 113549, 1, 9, 14}
+
+// OID and fixed value for the "must staple" variant of the TLS Feature
+// extension:
+//
+//  Features ::= SEQUENCE OF INTEGER                  [RFC7633]
+//  enum { ... status_request(5) ...} ExtensionType;  [RFC6066]
+//
+// DER Encoding:
+//  30 03 - SEQUENCE (3 octets)
+//  |-- 02 01 - INTEGER (1 octet)
+//  |   |-- 05 - 5
+var oidTLSFeature = asn1.ObjectIdentifier{1, 3, 6, 1, 5, 5, 7, 1, 24}
```

Comment:
@jsha That's the module identifier. Look at the IANA Considerations section; 1.3.6.1.5.5.7.1 is used to allocate the actual TLS Feature codepoint, but 1.3.6.1.5.5.7.0 is used to allocate a number for the ASN.1 module.